apps/client/app/modules/photo/index.tsx

```diff
@@ -28,10 +29,20 @@ import { getSizeStringForContent } from '@/lib/image-fns.ts'
 import { safeString } from '@/lib/strings.ts'
 import { useAuth } from '@/providers/auth/index.tsx'
 import { type loader } from '@/routes/photos.$slug.ts'
+import { EditTitleModal } from './components/edit-title-modal/index.tsx'
+
+
+
 
 export const PhotoModule = () => {
 	const { currentUser } = useAuth()
 	const { content } = useLoaderData<typeof loader>()
+	const [openModal, setOpenModal] = useState(false);
+
+	const toggleModal = () => {
+		setOpenModal(!openModal)
+	}
```

Comment:
Just a nitpick. We have a hook that does all this for you in one line.

```suggestion
        import { useDisclosure } from '@/hooks/use-disclosure.tsx' // you can import it like so
	const editTitleDialogState = useDisclosure()
```